### PR TITLE
[Add] Interactive Python Tutorial Website 

### LINKED ISSUE #606: [Add] Interactive Python Tutorial Website
### Site URL

https://www.learnpython.org/

### Description of the above provided site

learnpython.org is a free interactive Python tutorial - one of the Interactive Tutorials website. 
This website's vision is to teach coding within the browser using short and effective exercises. By running actual code directly from the web browser, learners will be able to try out coding without installing and executing it locally, which can be hard and redundant for the purpose of learning how to code.
This would be a great addition to the project.

### Category

Languages

### Sub Category

Python

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by aashish-aanand**
## Fixes Issue

Closes #606 
  
## Changes proposed

Added Python language tutorial in language section of database.

## Screenshots

image

## Note to reviewers
Please review my change and add it to your main branch

### CODE REVIEW

File: database/languages/python.json [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+  {
+    "name": "Learn Python 3",
+    "description": "Learn the basics of Python 3, one of the most powerful, versatile, and in-demand programming languages today.",
+    "url": "https://www.codecademy.com/learn/learn-python-3",
+    "category": "languages",
+    "subcategory": "python"
+  },
+  {
+    "name": "Python Tutorial",
+    "description": "Learn Python through interactive Python Tutorial and exercises and validate your skill with certificate.",
```
Comment (CBID2) [inline database/languages/python.json:11]:
```suggestion
    "description": "Learn Python through this interactive Python tutorial and exercises. You  also receive a certificate if you pass them.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @aashish-aanand! Aside from the typo I mention in my inline comment, you’re off to a great start! :) In the future, run through your descriptions in Grammarly or spell check before  adding them to your PR.

**Review by CBID2 — APPROVED** [review]
Great job @aashish-aanand! :) LGTM! :)

**Comment by aashish-aanand** [comment]
> Great job @aashish-aanand! :) LGTM! :) 

Thank you ma'am 😌

**Pull request merged**